configuration.json

```diff
@@ -0,0 +1,12 @@
+{
+	
+	"path_data": "./template_data",
+
+	"path_template": "./prelim_template",
+
+	"subjects": ["Harshman_Dobby", "Jacobsen_Petri"],
```

Comment:
what is this field supposed to mean? adding a documentation associated with your config file would be useful-- alternatively, use YAML format, which allows comments